src/style.css

```diff
@@ -0,0 +1,96 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 100px;
```

Comment:
better move typography styles to the `html{}` 😉

Reply:
Yep. You're right. My mistake.